.github/workflows/ansible-test.yml

```diff
@@ -11,19 +11,20 @@ jobs:
     name: Build collection
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        ansible: [2.9.27, 2.10.17, 2.11.12, 2.12.10, 2.13.5]
+        ansible: [v2.9.27, v2.10.17, v2.11.12, v2.12.10, stable-2.13, stable-2.14]
     steps:
     - name: Check out code
       uses: actions/checkout@v3
     
-    - name: Set up Python 3.8
+    - name: Set up Python 3.9
       uses: actions/setup-python@v4
       with:
-        python-version: 3.8
+        python-version: 3.9
```

Comment:
should/can we make this a variable so we can do easier change of python version next time we bump to 3.10?